frontend/src/store/index.js

```diff
@@ -284,6 +284,20 @@ export const store = new Vuex.Store({
       }
       return ''
     },
+
+    liaison_designate(state) {
```

Comment:
This function should probably return True or False, instead of 1 or ''. You could also update it's name to `is_liaison_designate` to be more verbose about what the getter is doing, but I'll leave that to you to decide.

Reply:
Same goes for the other function

Reply:
I'll cover this in a separate PR